about.html

```diff
@@ -0,0 +1,122 @@
+          to make new friends from around the world. Model UN is a valuable experience that can help you get
+           into college, find your first job, and become inspired to change the world.</p>
+      </div>
+      <div class="abot-div2">
+        <p>please contact us through email for further information and more about DCD FASHION .</p>
+        <p class="email">divinemaina@gmail.com</p>
+      </div>
+      <div class="abot-div3">
+        <div>
+          <h5>How do I prepare for my first participation in DCD fashion ?</h5>
+          <hr>
+          <p>Once you have signed up for a conference, you will receive your country assignment, your committee, and its topics. Most conferences provide a Background Guide or Topic Synopsis that introduces the topics
+
+            There are typically three items to prepare before you walk into your DCDfashion event: theproject you will present, your Opening Speech, and a Research Binder.</p>
+        </div>
+        <div class="about-logo">
+          <img class="about-logoimg" src="./assets/images/airbnb.png" alt="image" >
+        </div>
+        <div>
+          <h5>the past DCD fashion modeling event</h5>
+          <p>currently we have had more than seven worldwide event which were held in different countries </p>
+          <div class="events-pictures">
+            <div >
+            <div class="past-summit-overlay">
+              <h1 class="past-summit-overlay-title">2015</h1>
+              <p class="past-summit-overlay-paragraph">ladies night</p>
+            </div>
+          
+              <img  class="past-pictrs" src="./assets/images/modal3.JPG" alt="ghth" >
+            </div>
+            <div>
+              <div class="past-summit-overlay">
+                <h1 class="past-summit-overlay-title">2010
+                </h1>
+                <p class="past-summit-overlay-paragraph">gentleme's night</p>
+              </div>
+              <img  class="past-pictrs" src="./assets/images/modal5.JPG" alt="ghth" >
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="paterners">
+      <div>
+        <h5>paterners</h5>
+      </div>
+      <div class="patteners-imglogo">
+        <img  class="patterners-logo" src="./assets/images/airbnb.png" alt="" >
+        <img class="patterners-logo"  src="./assets/images/daum.png" alt="" >
+        <img class="patterners-logo"  src="./assets/images/naver.png" alt="" >
+        <img class="patterners-logo" src="./assets/images/.png" alt="" >
+        <img class="patterners-logo" src="./assets/images/.png" alt="" >
+      </div>
+     
+    </section>
+    <footer>
+      <div>
+      <img class="logo-image" src="./assets/images/logo1.png" alt="">
+      </div>
+      <p>DCD FASHION house our main priority is to increase confidence in youth where participants use their beauty ,brain and curture to solve some community's problem through the projects the do present 
+       .</p>
+    </footer>
```

Comment:
- [ ] Kindly change the background color and the text color of the Home Page footer, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),